deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/IncompatibleKerasConfigurationException.java

```diff
@@ -0,0 +1,12 @@
+package org.deeplearning4j.nn.modelimport.keras;
+
+
+public class IncompatibleKerasConfigurationException extends RuntimeException {
```

Comment:
Added Keras model import doc page with pithy explanation. Added link to that in javadoc comments and error message.